### DISCUSSION TITLE
[V4] 'adonis migration:run' does not ends

### DISCUSSION THREAD

**Discussion by wemersonrv** [adonisjs/core#3053] [Help] [answered]
Hello,

I am lhaving an issue with migration command. It just don't ends and i need to stop it manually:

```bash
$ adonis migration:run
Nothing to migrate
```

It stays here and does not ends. And this is  messing up my deploy script because it's parked at this point and after 10 minutes my deploy tool doesn't just gives me error and do not end the flow.

I know V4 is legacy now, and I'm redesigning the project on V5. But i'm not done yet and this issue is causing me problems. 

Does anyone have a tip on how to solve this on V4 for now?

**Comment by feniljariwala82** [comment, marked as answer]
Can you please remove the Lucid package and reinstall it?

**Reply by wemersonrv** [reply]
Hi. Updated and same problem happens!

I was using the `v6.2.1` and update it to `v6.3.0`. I think this is the latest version for Adonis v4  right?!

**Reply by wemersonrv** [reply]
Found it. I've just add a `process.exit()` at the end of Lucid migration commands and it works. 

i don't know if the repo is accepting PRs for legacy versions, but i will make it anyway  because it's important for me because i'm still using legacy Adonisjs v4, and i think others may be using it too.

**Reply by feniljariwala82** [reply]
That's great! I am happy that you found the error. But one thing I want to suggest to you and is "You should migrate to V5 now". I am just suggesting. It's up to you!

**Reply by wemersonrv** [reply]
> That's great! I am happy that you found the error. But one thing I want to suggest to you and is "You should migrate to V5 now". I am just suggesting. It's up to you!

Yeah. I'm migrating...  ooops, recreating from scratch 🤣  . But now i just need to run some changes on my legacy version until I finish the new version. And when run migrations found this problem.

**Reply by wemersonrv** [reply]
Add a PR: [https://github.com/adonisjs/lucid/pull/716](https://github.com/adonisjs/lucid/pull/716)

**Reply by e-e** [reply]
@wemersonrv I noticed that you marked this response as the answer, but also created the PR (which is currently closed). Did you find a solution that was different from that of your pull request?

**Reply by e-e** [reply]
never mind, [i found the problem](https://github.com/adonisjs/core/discussions/3443)
